versions.tf

```diff
@@ -1,7 +1,10 @@
 terraform {
-  required_version = "~> 0.12.0"
+  required_version = "~> 0.13.0"
 
   required_providers {
-    aws = "~> 2.70"
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
```

Comment:
I chose these version constraints since they seem most common in our other modules, but I'm open to changing this if we want to settle on a different standard.